server/xcode/xcode_darwin.go

```diff
@@ -151,6 +184,48 @@ func xcodePlistForPath(path string) (*xcodePlist, error) {
 	return &xcodePlist, nil
 }
 
+func commandLineToolsVersion(developerDir string) *xcodeVersion {
+	sdkPaths, err := fs.Glob(os.DirFS(developerDir), "SDKs/*")
+	if err != nil {
+		log.Warningf("Error reading CommandLineTools SDKs from %s: %s", developerDir, err.Error())
+		return nil
+	}
+	sdks := make(map[string]string)
+	for _, sdkPath := range sdkPaths {
+		sdks[strings.TrimSuffix(filepath.Base(sdkPath), ".sdk")] = sdkPath
+	}
+	return &xcodeVersion{
+		version:          defaultXcodeVersion,
+		developerDirPath: developerDir,
+		sdks:             sdks,
+	}
+}
+
+// Returns the version of the Xcode Command Line Tools, or "unknown" if it
+// can't be determined.
+func commandLineToolsVersionString() string {
+	out, err := exec.Command(
+		"sh", "-c",
+		"pkgutil --pkg-info=com.apple.pkg.CLTools_Executables | awk '/version:/ {print $2}'",
+	).Output()
+	if err != nil {
+		log.Warningf("Error determining Xcode Command Line Tools version: %s", err.Error())
```

Comment:
Same here, should I send the error up?

Reply:
Typically yes -- in this case it seems like you just log this version string for info only, so doesn't matter a ton, is that right?